### PR TITLE
chore: Add Guidance link from support form

### LINKED ISSUE #3882: Update Support form to link to Guidance help articles
Link people to the self-serve documentation within the support form.

### PR DESCRIPTION
**Pull request by anikbrazeau**
From #3259
Fixes #3882

### CODE REVIEW

File: i18n/translations/en/form-builder.json [hunk 1]
```diff
@@ -737,7 +737,9 @@
       "title": "Reason for reaching out"
     },
     "title": "Get support",
-    "useThisForm": "Use this form if you experience technical issues with GC Forms.",
+    "experience": "If you experience technical issues with GC Forms, ",
+    "guidanceLink": "consult our guidance",
+    "useThisForm": " or use this form to get in touch with our support team.",
```
Comment (anikbrazeau) [inline i18n/translations/en/form-builder.json:742]:
```suggestion
    "experience": "Explore our",
    "guidanceLink": "guidance articles",
    "useThisForm": " or get technical help from our team through this form.",
```

File: i18n/translations/fr/form-builder.json [hunk 2]
```diff
@@ -737,7 +737,9 @@
       "title": "Raison pour laquelle vous nous contactez"
     },
     "title": "Obtenir du soutien",
-    "useThisForm": "Utilisez ce formulaire pour obtenir du soutien pour tout problème technique avec Formulaires GC.",
+    "experience": "Pour tout problème technique avec Formulaires GC, ",
+    "guidanceLink": "consultez nos guides de références",
+    "useThisForm": " ou utilisez ce formulaire pour communiquer avec notre équipe de soutien.",
```
Comment (anikbrazeau) [inline i18n/translations/fr/form-builder.json:742]:
```suggestion
    "experience": "Explorer nos ",
    "guidanceLink": "guides de références",
    "useThisForm": " ou obtenez du soutien de notre équipe en remplissant ce formulaire.",
```

File: i18n/translations/en/form-builder.json [hunk 3]
```diff
@@ -737,7 +737,9 @@
       "title": "Reason for reaching out"
     },
     "title": "Get support",
-    "useThisForm": "Use this form if you experience technical issues with GC Forms.",
+    "experience": "Explore our",
```
Comment (anikbrazeau) [inline i18n/translations/en/form-builder.json:740]:
```suggestion
    "experience": "Explore our ",
```

File: app/(gcforms)/[locale]/(support)/support/components/client/SupportForm.tsx [hunk 4]
```diff
@@ -54,7 +54,11 @@ export const SupportForm = () => {
       )}
 
       <h1>{t("support.title")}</h1>
-      <p className="mb-6 mt-[-2rem] text-[1.6rem]">{t("support.useThisForm")}</p>
+      <p className="mb-6 mt-[-2rem] text-[1.6rem]">
+        {t("support.experience")}
+        <Link href={`https://articles.alpha.canada.ca/forms-formulaires/${language}/guidance`}>{t("support.guidanceLink")}</Link>
```
Comment (anikbrazeau) [inline app/(gcforms)/[locale]/(support)/support/components/client/SupportForm.tsx:59]:
```suggestion
        <Link href={`https://articles.alpha.canada.ca/forms-formulaires/${language}/guidance` target=\"_blank\"}>{t("support.guidanceLink")}</Link>
```

File: app/(gcforms)/[locale]/(support)/support/components/client/SupportForm.tsx [hunk 5]
```diff
@@ -54,7 +54,11 @@ export const SupportForm = () => {
       )}
 
       <h1>{t("support.title")}</h1>
-      <p className="mb-6 mt-[-2rem] text-[1.6rem]">{t("support.useThisForm")}</p>
+      <p className="mb-6 mt-[-2rem] text-[1.6rem]">
+        {t("support.experience")}
+        <Link href={"https://articles.alpha.canada.ca/forms-formulaires/${language}/guidance" target=\"_blank\"}>{t("support.guidanceLink")}</Link>
+        {t("support.useThisForm")}
```
Comment (anikbrazeau) [inline app/(gcforms)/[locale]/(support)/support/components/client/SupportForm.tsx:60]:
```suggestion
        <Link href={`https://articles.alpha.canada.ca/forms-formulaires/${language}/guidance` target=\`_blank\`}>{t("support.guidanceLink")}</Link>
        {t("support.useThisForm")}
```

**Review by thiessenp-cds — APPROVED** [review]
LGTM :)

**Pull request merged**